decred/decred/util/database.py

```diff
@@ -212,7 +212,7 @@ def __contains__(self, k):
         cursor = self.conn.cursor()
         cursor.execute(self.existsQuery, (k,))
         row = cursor.fetchone()
-        if row is None:
+        if row is None:  # nocover
```

Comment:
If we can hit this in testing, we should.

Reply:
According to the SQLite docs, we cannot (see [The EXISTS operator](https://sqlite.org/lang_expr.html#exists_op) and [The count(X) function](https://sqlite.org/lang_aggfunc.html#count)).

I didn't remove these checks in case the SQLite behavior changes, or its docs are inaccurate, or we end up supporting another relational database.